README.md

```diff
@@ -193,6 +193,15 @@ for doing very specific things.
 | load the analyzer tool on a stats file                                                                      | `yarn analyze`                                                                                                                                                                                                               |
 | add a new React app                                                                                         | `yarn new:app` (make sure you have [`vagov-content`](https://github.com/department-of-veterans-affairs/vagov-content/) sibling to `vets-website`)                                                                            |
 
+## Debugging React with VS Code
+
+1. Install [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) extension.
+1. Start development server: `yarn watch`
+1. Set a breakpoint in VS Code
+1. Run "Lauch Chrome Debugger" from VS Code debug panel
+
+ [_Reference_](https://code.visualstudio.com/docs/nodejs/reactjs-tutorial#_debugging-react)
+
```

Comment:
Can we put this in the `docs/` folder?

Reply:
sure! would it be worth linking to `/docs` from `README.md`? i didn't notice that dir existed before

Reply:
ok i moved the react stuff and linked from main README, also tried to cleanup old/outdated files